pkg/k8s/watchers/cilium_endpoint.go

```diff
@@ -262,3 +260,14 @@ func CreateCiliumEndpointLocalPodIndexFunc() cache.IndexFunc {
 		return indices, nil
 	}
 }
+
+// Helpers...
+
+func (k *K8sWatcher) updatePromMX(cep *types.CiliumEndpoint) {
+	if cep == nil {
+		return
+	}
+	if p := k.endpointManager.LookupPodName(k8sUtils.GetObjNamespaceName(cep)); p != nil {
```

Comment:
Do we need to be concerned about additional updates triggering this? Is there some edge / state transition we can filter on so that we only record metrics for when the CEP is first configured?

Do we have to worry about this metric being bumped when the agent process restarts?

My worry is that if the CiliumEndpoint is updated a second time for some other reason, this will cause wildly inaccurate metrics, as that update event will be a long time after the pod creation time.